src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Connection.cs

```diff
@@ -316,7 +324,16 @@ void IConnectionControl.Pause()
         void IConnectionControl.Resume()
         {
             Log.ConnectionResume(ConnectionId);
-            _socket.ReadStart(_allocCallback, _readCallback, this);
+            try
+            {
+                _socket.ReadStart(_allocCallback, _readCallback, this);
+            }
+            catch (UvException ex)
+            {
+                // ReadStart() can throw a UvException in some cases (e.g. socket is no longer connected).
+                // This should be treated the same as OnRead() seeing a "normalDone" condition.
```

Comment:
OK, I will stop passing the exception, but I don't think logging the exception is correct either, since this is a perfectly normal thing to happen, and logging an exception will confuse customers.

`Connection.OnRead()` just calls `Log.ConnectionReadFin()` for `normalDone`.  Should I do the same in Resume()?